code/datums/controllers/process/cross_server_sync.dm

```diff
@@ -0,0 +1,13 @@
+/// proocess for flock_structure objects
+/datum/controller/process/cross_server_sync
+	setup()
+		name = "Cross Server Sync"
+		schedule_interval = 1 MINUTE
+
+#ifdef LIVE_SERVER
+		for(var/datum/game_server/game_server as anything in game_servers.servers)
+			game_server.sync_server_data()
```

Comment:
I don't mess with these things, but is this not just "do it immediately as well as when the scheduler says so"?
```suggestion
		src.doWork()
```

Reply:
yes, because loops dont start running until round start but they do setup